config/peerpods/podvm/lib.sh

```diff
@@ -196,11 +196,18 @@ function prepare_source_code() {
             error_exit "Failed to enable fips mode"
     fi
 
-    if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
+    # links must be relative
+    if [[ -e /tmp/kata-opa/custom.rego ]] ; then
+        echo "Setting custom agent policy according to file found"
+        cp /tmp/kata-opa/custom.rego "${podvm_dir}"/files/etc/kata-opa/custom.rego
+        ln -sf custom.rego  "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
+    elif [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]] ; then
+        echo "Setting custom agent policy to CoCo's recommended policy"
         sed 's/default SetPolicyRequest := true/default SetPolicyRequest := false/; s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
             "${podvm_dir}"/files/etc/kata-opa/default-policy.rego > "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
-        ln -sf "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego  "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
+        ln -sf coco-default-policy.rego  "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
     fi
+    echo "\nCurrent Agent Policy:\n" && cat "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
```

Comment:
Hmm afaik echo will not expand the escape sequences. `printf` needed ?

Reply:
While you are touching this file, it might be useful to split the sed command into multiline for easier readability. for example

```
  sed 's/default SetPolicyRequest := true/default SetPolicyRequest := false/;
   s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
        "${podvm_dir}"/files/etc/kata-opa/default-policy.rego > \
        "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
```
Btw on second thoughts should SetPolicyRequest be set to true for now ?

Reply:
So as default coco policy, we can disable `ReadStreamRequest` and `ExecProcessRequest` to start with and provide SetPolicy examples to enable/disable apis for trying out different scenario
Also we'll document (blog) the (wip) eventual solution with initData and measured disk. 
Sounds ok ?

Reply:
> Btw on second thoughts should SetPolicyRequest be set to true for now ?

Newbie question : what does SetPolicyResquest do ?

> So as default coco policy, we can disable `ReadStreamRequest` and `ExecProcessRequest` to start with and provide SetPolicy examples to enable/disable apis for trying out different scenario

IIUC this is calling for another PR that re-enables `SetPolicyRequest` and disables `ReadStreamRequest`, right ?

Reply:
A new PR or at least a new commit explaining the rationale.